lambdas/build-cri-oauth-request/build.gradle

```diff
@@ -7,7 +7,6 @@ plugins {
 
 dependencies {
 	implementation libs.bundles.awsLambda,
-			libs.diVocab,
```

Comment:
(added as an api dependency in `common-services`, so this is now available anywhere that uses the common lib)